packages/eslint-config/rules/style.js

```diff
@@ -0,0 +1,775 @@
+module.exports = {
+  rules: {
+    /**
+     * enforce linebreaks after opening and before closing array brackets
+     */
+    'array-bracket-newline': ['off'],
+
+
+    /**
+     * enforce consistent spacing inside array brackets
+     */
+    'array-bracket-spacing': ['error', 'never'],
+
+
+    /**
+     * enforce line breaks after each array element
+     */
+    'array-element-newline': ['off'],
```

Comment:
same as above should we consider making this consistent with objects?

Reply:
with the same minimum 5 properties rule, I agree

Reply:
Turns out... we cannot be consistent with object rules. It is either too strict or too loose, and the feature request that adds what we want got ignored by the ESLint team. ಠ_ಠ

When it's too strict, anything that is under 5 elements MUST be single line or include a single element that's multi-line. When it's too loose, all we get is consistency in how newlines after opening / before closing brackets are handled and not the elements themselves.... Since we cannot be consistent I'm electing to leave these rules off so that it can be set by the project using the configs if they wish.